curl-static-musl/plan.sh

```diff
@@ -0,0 +1,60 @@
+source ../curl/plan.sh
+
+pkg_name=curl-static-musl
+pkg_distname=curl
+pkg_origin=core
+pkg_version=7.51.0
+pkg_description="curl is an open source command line tool and library for
+  transferring data with URL syntax."
+pkg_upstream_url=https://curl.haxx.se/
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('curl')
+pkg_source=https://curl.haxx.se/download/${pkg_name}-${pkg_version}.tar.gz
```

Comment:
This needs to be `pkg_distname` instead of `pkg_name`, otherwise it can't download.